newIDE/app/src/MainFrame/index.js

```diff
@@ -396,7 +393,7 @@ const MainFrame = (props: Props) => {
     [state.currentProject, state.currentFileMetadata]
   );
 
-  const openFromFileMetadata = (fileMetadata: FileMetadata): Promise<void> => {
+  const openFromFileMetadata = (fileMetadata: FileMetadata): Promise<any> => {
```

Comment:
`void` is better than `any` (void means nothing returned, while any can accept anything). Can we go back to `void`?